OOP.Task4/Program.cs

```diff
@@ -44,22 +44,22 @@ static string InputAndValidation()
                     Input.ToCharArray();
 
                     char temp = Input[0];
-                    int counter = 0;
+                    int count = 0;
 
 
                     for (int i = 0; i < Input.Length - 1; i++)
                     {
                         if (Input[i + 1] == temp)
                         {
-                            counter++;
+                            count++;
                         }
                     }
 
                     if (Input.Length < 3)
                     {
                         Console.WriteLine("Введите слово состоящее из трех и более букв");
                     }
-                    else if (counter != Input.Length - 1)
+                    else if (count != Input.Length - 1)
                     {
                         if (IsPalindrome(Input) == true)
```

Comment:
I don't think the code that starts from 64th row relates to InputAndValidation() method. could you please move it to a separate method (e.g. to Main)?